properties/border/radius.json

```diff
@@ -0,0 +1,24 @@
+{
+  "border": {
+    "radius": {
+      "none": {
+        "value": "0"
+      },
+      "s": {
+        "value": "4px"
+      },
+      "m": {
+        "value": "8px"
+      },
+      "l": {
+        "value": "16px"
+      },
+      "circle": {
+        "value": "100%"
+      },
+      "pill": {
+        "value": "160px"
```

Comment:
Esse valor me parece um pouco alto para o `pill`. Na web `30px` é o suficiente. Só tenho dúvida com relação as outras plataformas..

Reply:
A ideia é utilizar esse Token em botões para deixá-los com as bordas completamente arredondadas (com o formato de uma pílula mesmo). Qualquer border-radius maior que a altura do botão já dá essa aparência, então podemos reutilizar esse Token independentemente do botão em que ele for aplicado, já que nenhum botão será maior do que 160px. O que acha?

Reply:
Para isso acho que serve usar o valor de `50%`.

Reply:
Testei e não deu certo 😢
Olha só como o botão ficou:
![image](https://user-images.githubusercontent.com/22939260/61566238-dbe80680-aa51-11e9-8719-a4fafea73e2a.png)

Reply:
Com pixels fica com esses testes?

Reply:
@pyymenta assim:
![image](https://user-images.githubusercontent.com/22939260/62053092-497ff980-b1ed-11e9-9037-7f0ba0e25561.png)

Reply:
Com pixels fica mais próximo do esperado.. O que acha?

Reply:
Concordo! Você sugeriu 30px no seu primeiro comentário, mas se em algum momento nós precisarmos de algum botão com altura maior do que 60px, esse valor já deixaria de funcionar... Acho melhor algum valor maior do que 30px, só por segurança.

Reply:
bom, então nesses casos acho válido sim colocar um tamanho maior, 160px deve suprir bem

Reply:
Fechou! Valeu @pyymenta